### PR TITLE
Unit test and documentation

### PR DESCRIPTION
**Pull request by SyntaxXeror**
Small update to fix some of the issues from issue #18 : 
* Add user instructions in docs/CLIENT.md
* remove warning for react grid test

### CODE REVIEW

File: client/test/DigitalTwins.test.tsx [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
-  it('Reder OK', () => {
-    render(
-        <DigitalTwins />
-    );
+  it('reder DigitalTwin', () => {
+    render(<DigitalTwins />);
     expect(true);
   });
+
+  it('renders Dashboard woth components', () => {
```
[the remark this answers is not in the document]
Comment (SyntaxXeror) [reply client/test/DigitalTwins.test.tsx:15]: @prasadtalasila So codeclimate is kind of dumb when it comes to test... Is it okay if we exclude test from codeclimate or how do we prevent this redunt code smell?

**Comment by prasadtalasila** [comment]
Please exclude

File: client/test/DigitalTwins.test.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (prasadtalasila) [reply client/test/DigitalTwins.test.tsx:20]: @SyntaxXeror 
Please check codeclimate config docs. If I am not mistaken, there must be a way to skip this problem.

[the remark this answers is not in the document]
Comment (SyntaxXeror) [reply client/test/DigitalTwins.test.tsx:20]: @prasadtalasila you already replyed to this early where you wrote directly from email, so this is fixed :)

**Pull request merged**